server/schemas/typeDefs.js

```diff
@@ -7,7 +7,7 @@ const typeDefs = gql`
     type User {
         _id: ID
         business_id: ID
-        business_brand_name: String
+        brand_name: String
```

Comment:
Updating to match already existing name so easier to work with everything and removing complexity